### DISCUSSION TITLE
Preventing a part of an expression from incrementing, when copied with Pc.

### DISCUSSION THREAD

**Discussion by rolf-stargate** [andmarti1424/sc-im#600] [Q&A] [answered]
Hi,
I only wanted to ask if it is possible to prevent a cell reverence in an expression from incrementing, when I copy it with the Pc command? For example copying B1*A1 and when past in again, on a different row, B1 becomes, lets say B5 but A1 stays the same. I could't find anything in the docs, but maybe I missed it.
Awesome program by the way!! I love it!

**Comment by andmarti1424** [comment, marked as answer]
hello. Did you try referencing it like this: `$A$1` ?

**Reply by rolf-stargate** [reply]
Thank you very much! That worked perfectly!
